go.mod

```diff
@@ -14,6 +14,7 @@ require (
 
 require (
 	github.com/matryer/moq v0.5.3 // indirect
+	github.com/octavore/delta v0.6.1
```

Comment:
You'll probably want to remove this and redo `go mod tidy`; it should be in the section above but sometimes it doesn't behave.